sumo-api/sumo/metrics/evaluator.py

```diff
@@ -86,7 +86,140 @@ def default_settings():
 
         thresholds = np.linspace(0.5, 0.95, 10)
         recall_samples = np.linspace(0, 1, 11)
-        categories = ["wall", "chair"]
```

Comment:
We should load this from a .txt/.csv file.

Reply:
Could do, but then the software has an additional file dependency to make it work.